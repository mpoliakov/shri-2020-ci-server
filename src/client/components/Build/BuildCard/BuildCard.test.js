import React from 'react';
import renderer from 'react-test-renderer';

import BuildCard from './BuildCard';
import stubBuild from '@stub/build'

it(`<BuildCard/> is rendered correctly`, () => {
  const component = renderer.create(<BuildCard film={stubBuild}/>).toJSON();
  expect(component).toMatchSnapshot();
});
