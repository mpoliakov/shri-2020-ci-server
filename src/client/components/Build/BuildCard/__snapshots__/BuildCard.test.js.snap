// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BuildCard/> is rendered correctly 1`] = `null`;
